test/e2e/suite/conformance/certificates/suite.go

```diff
@@ -73,7 +77,11 @@ func (s *Suite) complete(f *framework.Framework) {
 	//Expect(s.CreateIssuerFunc).NotTo(BeNil(), "CreateIssuerFunc must be set")
 
 	if s.DomainSuffix == "" {
-		s.DomainSuffix = f.Config.Addons.IngressController.Domain
+		if s.DomainSuffixType == "istio" {
+			s.DomainSuffix = f.Config.Addons.Istio.Domain
+		} else {
+			s.DomainSuffix = f.Config.Addons.IngressController.Domain
+		}
```

Comment:
We should do a switch here, and error on default (unrecognised suffix type).

Reply:
Done